testing/integration/src/tester.rs

```diff
@@ -173,11 +173,11 @@ impl Tester {
     }
 
     /// Get blockstore
-    pub fn blockstore(&'static self) -> Box<dyn Blockstore + 'static> {
+    pub fn blockstore(&self) -> &dyn Blockstore {
         if self.executor.is_some() {
-            Box::new(self.executor.as_ref().unwrap().blockstore())
+            self.executor.as_ref().unwrap().blockstore()
         } else {
-            Box::new(self.state_tree.store())
+            self.state_tree.store()
```

Comment:
I simplified this return type to dyn. In practice this means that we can't call some generic methods with the constraint `Self: Sized`, but nothing is calling this still, so we'll cross that bridge when we get to it.

A nicer solution here would be to use a typestate pattern (see #4) to restrict which methods are available on uninitialized and initialized Testers. But let's save that for another day.